gitzones/sshd_config

```diff
@@ -0,0 +1,24 @@
+# Auth & security settings
+PasswordAuthentication no
+ChallengeResponseAuthentication no
+AuthenticationMethods publickey
+PermitRootLogin no
+AllowUsers git
+
+# Features
+AllowAgentForwarding no
+AllowTcpForwarding no
+X11Forwarding no
+PermitTTY no
+PrintMotd no
+
+# Logging
+LogLevel VERBOSE
```

Comment:
How much logging does that produce per connection?

Reply:
A lot, but it's also the least verbose level that logs the key fingerprint used for log in. Hence it's recommended by Mozilla. I guess we don't need that? It's not like we do not expect our authorized keys to *not* login.

Reply:
Set to `INFO`, which is also the default